hms-app/src/components/input-forms/IdeaForm.tsx

```diff
@@ -300,8 +300,8 @@ function IdeaForm(props: IProps) {
 
             <Card.Section className={classes.borderSection}>
               <Checkbox.Group
-                label='Required skills'
-                description='chose one or more required skills'
+                label='Skills'
+                description='Select one or more skills that could help to bring your idea to life'
```

Comment:
can we use Choose here also? 
so that we not using "Select" for skills and "Choose" for categories :)